dtcli/scripts/dt.py

```diff
@@ -135,9 +161,21 @@ def sanitize_url(ctx, param, value):
                            )
 
 
-tenant_url = click.option(
+def tenant_error_handler(func):
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except requests.exceptions.ConnectionError as e:
+            err = re.sub(r'\<[\w\s\d\.]+\>:\s|\[[\w\d\s\-]+\]\s', '',str(e.args[0].reason))
+            # TODO Extract to generic handler
+            raise click.UsageError(f"{err}")
```

Comment:
Not sure about the UsageError after all - I mean it can be either the user or the environment... We can maybe expand this message